### PR TITLE
[docs](docs)update en docs

### PR DESCRIPTION
**Pull request by httpshirley**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by luzhijing** [comment]
LGTM

File: docs/en/docs/lakehouse/multi-catalog/hive.md [hunk 1]
```diff
@@ -26,4 +26,149 @@ under the License.
...
+    'dfs.nameservices'='your-nameservice',
+    'dfs.ha.namenodes.your-nameservice'='nn1,nn2',
+    'dfs.namenode.rpc-address.your-nameservice.nn1'='172.21.0.2:4007',
+    'dfs.namenode.rpc-address.your-nameservice.nn2'='172.21.0.3:4007',
+    'dfs.client.failover.proxy.provider.your-nameservice'='org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider'
+);
+	
+# 2. Create Catalog and use an existing Resource. The key and value information in the followings will overwrite the corresponding information in the Resource.
+CREATE CATALOG hive WITH RESOURCE hms_resource PROPERTIES(
+	'key' = 'value'
```
Comment (morningman) [inline docs/en/docs/lakehouse/multi-catalog/hive.md:133]:
```suggestion
    'key' = 'value'
```

File: docs/en/docs/lakehouse/multi-catalog/multi-catalog.md [hunk 2]
```diff
@@ -261,7 +261,7 @@ See [Hudi](./hudi)
 
 ### Connect to Elasticsearch
 
-See [Elasticsearch](./elasticsearch)
+See [Elasticsearch](./es)
```
Comment (morningman) [inline docs/en/docs/lakehouse/multi-catalog/multi-catalog.md:264]: Modify the Chinese version too

File: docs/en/docs/lakehouse/multi-catalog/hive.md (hunk 1 above)
Comment (morningman) [reply docs/en/docs/lakehouse/multi-catalog/hive.md:133]: Please use 4 spaces to replace all tabs

Comment (morningman) [reply docs/en/docs/lakehouse/multi-catalog/hive.md:133]: Please use 4 spaces to replace all tabs

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**